.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Python: Current File",
+            "type": "python",
+            "request": "launch",
+            "program": "${file}",
+            "console": "integratedTerminal"
+        }
+    ]
+}
```

Comment:
*For future reference, we'll be adding this file into a .gitignore. A .gitignore is a file that prevents specific files into being commited+pushed into a remote repo. Files included in .gitignore include code editor settings, environment files, virtual env folder, etc. Basically any file(s) specific to an individual's coding setup.